packages/block-library/src/list-item/hooks/use-outdent-list-item.js

```diff
@@ -112,7 +112,7 @@ export default function useOutdentListItem( clientId ) {
 					getBlockIndex( parentListItemId ) + 1
 				);
 				if ( ! getBlockOrder( parentListId ).length ) {
-					removeBlock( parentListId );
+					removeBlock( parentListId, false );
```

Comment:
```suggestion
					const shouldSelectParent = false;
					removeBlock( parentListId, shouldSelectParent );
```

Reply:
I left these suggestions to make the code easier to grok...but feel free to ignore them.